test/src/scripts/main.spec.ts

```diff
@@ -5,15 +5,15 @@ import { describe, expect, test, jest, afterEach, beforeEach } from '@jest/globa
 
 import { main } from "../../../src/scripts/main";
 import { saveToFile } from "../../../src/utils/fs";
-import { getGitOrgData, setTeamsData } from "../../../src/utils/index";
+import { getOrgData, getTeamsData } from "../../../src/utils/index";
 
 import { MOCK_ORGANIZATION } from '../../mock/repos_info';
-import { REPOS_FILE_PATH, TEAMS_FILE_PATH } from '../../../src/config';
+import { REPOS_FILE_PATH } from '../../../src/config';
 
 const spyConsoleError = jest.spyOn(console, 'error');
 
-const mockGetGitOrgData = getGitOrgData as jest.Mock;
```

Comment:
Should `setOrgData()` inside `main.ts` also be mocked and checked that it's called?